extra/3-magic-8-ball.js

```diff
@@ -45,8 +45,34 @@
 
 // This should log "The ball has shaken!"
 // and return the answer.
+
 function shakeBall() {
-  //Write your code in here
+  expectedAnswers = [
```

Comment:
This variable is missing a const or let declaration

Reply:
Thank you Kate .As you can see , I declared it first outside of the function which makes it global . Then I changed my mind  and moved it to the inside of the function without declaration , just trying to make it global from the inside of the function to be able to use it in the other function . tried both ways and both work .